src/main/resources/languages/zh-CN.json

```
@@ -28,8 +28,8 @@
         "login_token_attempt": "[Dispatch] 客户端 %s 正在尝试通过 token 登录",
         "login_token_error": "[Dispatch] 客户端 %s 通过 token 登录失败",
         "login_token_success": "[Dispatch] 客户端 %s 已通过 token 登录，UID 为 %s",
-        "login_password_error": "🇺🇸[Dispatch] Client %s failed to log in via password.",
-        "login_password_storage_error": "🇺🇸[Dispatch] Client %s failed to log in via password because there is no password in the database.",
+        "login_password_error": "🇺🇸[Dispatch] 客户端 %s 无法通过密码登录",
+        "login_password_storage_error": "🇺🇸[Dispatch] 客户端 %s 无法登录！ 因为数库中没有密码",
```

```suggestion
        "login_password_error": "[Dispatch] 客户端 %s 无法通过密码登录",
        "login_password_storage_error": "[Dispatch] 客户端 %s 无法登录！ 因为数库中没有密码",
```
Remove these when you update the strings :)